jenkins/buildall.cmd

```diff
@@ -7,6 +7,6 @@
 setlocal
 
 set JENKINS_BUILD=True
-call test\jenkins.testall.cmd %*

```

Comment:
No problem here, I just think it's goofy that Git/GitHub detected these two files as renames of each other. I guess the diff is the same size regardless of whether the script name changed.